docker-build-images/centos7-build.Dockerfile-ppc64le

```diff
@@ -0,0 +1,22 @@
+FROM ppc64le/centos:7
+MAINTAINER Shaun Crampton <shaun@tigera.io>
+ENV STREAM el7
+
+ARG UID
+ARG GID
+
+# Some commands that would typically be run at container build time must be run in a privileged container.
```

Comment:
Can you explain this a bit more?  I've never had privilege issues at build time before, don't builds run as root (I thought that was the point!)?

I wonder if now would be a good time to switch to the `entrypoint` approach used in https://github.com/projectcalico/go-build since that doesn't require a rebuild of the container to add the user.